alacritty/src/cli.rs

```diff
@@ -138,8 +140,27 @@ impl Options {
                     .takes_value(true)
                     .help("Override configuration file options [example: cursor.style=Beam]"),
             )
+            .subcommand(
+                SubCommand::with_name("msg").about("Send a message to the Alacritty socket").arg(
+                    Arg::with_name("create-window")
+                        .long("create-window")
+                        .help("Create a new window in the same Alacritty process")
+                        .required(true)
+                ),
+            )
             .get_matches();
 
+        // Handle `msg` subcommand.
+        if let Some(_matches) = matches.subcommand_matches("msg") {
```

Comment:
I'm currently working on migrating Alacritty to `structopt`, I think realistically that's the only way to merge this, since we want the `msg` subcommand to only be available on unix and there's no great way to do that with `clap` alone.

So I've left this part open until then mostly.